src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -286,6 +286,10 @@ WrapEncodedAudioIntoPCM =
 # Default: true
 MuxLPCMToMpeg = 
 
+# Output silent audio when a movie do not have audio in ffmpegvideo. Some of renderers do not support a movie without audio. This can be perfomed by either UMS.conf or renderer.conf.
+# Default: false
+OutputSilentAudio=
```

Comment:
again space, and please add some line breaks so it matches the style of the rest of the file